kateryna-tolochko/README.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started with AliExpress
+
+AliExpress is a popular online marketplace where you can find a wide variety of products at affordable prices. Whatever you are looking for, AliExpress offers a huge selection from sellers around the world. If you are new to AliExpress, this guide will help you get started and make the most of your shopping experience.
+
+## Step 1: Create an Account
+
+1. Visit the [AliExpress website](https://www.aliexpress.com/).
+
+2. Find **Join Free** button and click it.
+
+   ![ ](https://i.ibb.co/cyYxdZX/screen.png)
+
+3. After clicking, you should see a registration form and different options for signing up through social media accounts. Select your preferred method.
+
+4. If you chose to sign up through your social media accounts, you will be redirected to a page where you will need to give AliExpress permission to access your account information. Follow the on-screen instructions to complete the process.
+
+5. If you choose to sign up using your email, fill out the form. Input your valid email, password (6-20 symbols) and finally click **Create Account**.
+
+6. Check your email box and find the letter from AliExpress. Click the verification link provided in the email to verify your account. If you did not receive email with account verification link, please check your spam folder.
+
+7. Once your account is successfully verified, you will be directed to the AliExpress website. You will be asked to provide additional information, such as your name and shipping address. Fill in the required fields accurately.
+
+## Step 2: Browse Products
+
+Once you have created an account and logged in, you can start browsing for products.
```

Comment:
2 list items below assume a semicolon at the end of this sentence, or there's no need to use the list—just write the list items as 2 separate sentences.